src/References.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import styled from 'styled-components';
+import firebase from 'firebase/app';
+import { Data } from './Data';
+import { device } from './device';
+import { Link } from 'react-router-dom';
+export default function References() {
+    const name = firebase.auth().currentUser ? firebase.auth().currentUser.displayName : "Hello"
+    console.log(Data, 'Data')
+    console.log(Data[0], 'first')
+    const Scriptures = Data;
+    console.log(Scriptures, 'scrree')
+    return (
+        <div>
+            <NavBar>
+                        <Link style={{margin: "3px", color: "#FFF5EE"}} to="/activate">Activate</Link>
+                        <Link style={{margin: "3px", color: "#fffafa"}} to="/follow">Follow</Link>
+                        {firebase.auth().currentUser ?
+                        <Link style={{margin: "3px", color: "#fffafa"}} onClick={() => firebase.auth().signOut()}>SignOut</Link> :
```

Comment:
A Link component must have a to prop i.e `<Link to='/'>`
In this case I would suggest you use a div instead of a Link here then,
create a signOutHandler method i.e.
```
const signOutHandler = () => {
  firebase.auth().signOut().then(() => history.push("/"))
};
```

then on the sign out div/button you will use 
```
<div onClick={signOutHandler}>Sign Out</div>
```